lib/deadletterProcessors/reenqueueByName.js

```diff
@@ -0,0 +1,12 @@
+const deadletterActions = require('./../deadletterActions');
```

Comment:
minor `reenqueueAllByName`

Reply:
I thought about it but seemed misleading. so I removed the `All` although I can be argued eitherways. don't feel too strongly about it.

Reply:
nope, its a minor comment